spec/furniture/marketplace/collecting_payments_system_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+# @see https://github.com/zinc-collective/convene/issues/1576
+describe "Marketplace: Collecting Payments", type: :system do
+  # @see https://github.com/zinc-collective/convene/issues/1622
+  describe "via Stripe" do
+    it "Connecting a Vendor Stripe Account" do
```

Comment:
Nit: Possibly slightly better test case naming:
```suggestion
  describe "with Stripe" do
    it "connects a Vendor Stripe Account" do
```

Reply:
I guess the test right now actually doesn't test that it "connects" the Stripe account. More like it "prompts to connect" or "gives option to connect" or something.

Reply:
Yea, I was trying to figure out how to do the "actual test connection" bit but I ran out of spoons. Maybe someday!